api/CSSAnimation.json

```diff
@@ -14,10 +14,10 @@
             "version_added": "84"
           },
           "firefox": {
-            "version_added": false
+            "version_added": "75"
```

Comment:
Checks out, I think this was included in https://bugzilla.mozilla.org/show_bug.cgi?id=1619821#c12, part of https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/75#Web_animations_API.